packages/celotool/src/e2e-tests/utils.ts

```diff
@@ -48,21 +48,19 @@ export async function waitToFinishSyncing(web3: any) {
   }
 }
 
-export function assertAlmostEqual(
-  actual: BigNumber,
-  expected: BigNumber,
-  delta: BigNumber = new BigNumber(10).pow(12).times(5)
-) {
+// TODO(joshua): Pick a good percentage that works and is acceptably close
+export function assertAlmostEqual(actual: BigNumber, expected: BigNumber, pct: number = 0.5) {
```

Comment:
This seems awfully more lenient than the original, can we make this check more precise?

Reply:
reverted